api/migrations/alembic.ini

```diff
@@ -8,10 +8,9 @@
 # the 'revision' command, regardless of autogenerate
 # revision_environment = false
 
-
 # Logging configuration
 [loggers]
-keys = root,sqlalchemy,alembic
+keys = root,sqlalchemy,alembic,flask_migrate
```

Comment:
This is configuration that got added when I ran `flask db init`: https://flask-migrate.readthedocs.io/en/latest/index.html#example

I think this just sets up logging for `flask_migrate` in Alembic, but to be honest I'm not totally sure.